addons/sale/i18n/sale.pot

```diff
@@ -3374,6 +3366,7 @@ msgstr ""
 #: model:ir.model.fields.selection,name:sale.selection__product_document__attached_on__quotation
 #: model:ir.model.fields.selection,name:sale.selection__sale_order__state__draft
 #: model:ir.model.fields.selection,name:sale.selection__sale_report__state__draft
+#: model:ir.model.fields.selection,name:sale.selection__sale_subscription_report__state__draft
```

Comment:
This shows that you exported the terms with `sale`and `sale_subscription` installed in your database.
The best way to export translations is when there is only the required module installed ;).

The diff you have in the `.pot` should not be as big and related to your changes.